apps/explorer/src/modules/uncles/components/UnclesTableTitle/UnclesTableTitle.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+  <div class="block-title">
+    <div class="wrap" :class="pageType != 'home' ? 'pagination' : ''">
+      <div v-if="pageType == 'home'" class="the-title">{{ $t('block.last') }}</div>
+      <div v-else class="the-title">{{ title }}</div>
+      <a v-if="pageType == 'home'" class="button-link" href="/blocks"
+        ><button>{{ $t('btn.view-all') }}</button></a
+      >
+      <slot v-if="pageType != 'home'"></slot>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+export default {
```

Comment:
Let's try to use [vue-class-componnent](https://github.com/vuejs/vue-class-component) syntax to unify with other components

Reply:
I'll take a look at it.